getting_started.rst

```diff
@@ -378,11 +378,11 @@ For example, if we have a profile with a 32-bit GCC configuration in a profile c
     We strongly recommend using :ref:`profiles` and managing them with :ref:`conan_config_install`.
 
 However, the user can always override the default profile settings in the :command:`conan install` command using the :command:`--settings`
-parameter. As an exercise, try building the Encrypter project 32-bit version:
+parameter. As an exercise, try building the Encrypter project 32-bit version, after creating a new profile (gcc_x86):
```

Comment:
I agree, this part is very confusing, but I don't know if we fix it only with that change.

Reply:
I think the confusion comes from the name of the profile. Will try to push some changes here. Thanks for noticing @Titonus!